File: src/Beneficios.Domain/TenantDefaultUser.cs
namespace Beneficios.Domain;

public static class TenantDefaultUser
{
    public const string Nome = "Usuário cadastrar";
    public const string Email = "user@123.com";
    public const string Senha = "user@123";

    public static bool IsDefaultUser(string email) =>
        string.Equals(email.Trim(), Email, StringComparison.OrdinalIgnoreCase);
}

File: Beneficios.Front/src/app/shared/constants/tenant-default-user.ts
export const TENANT_DEFAULT_USER = {
  nome: 'Usuário cadastrar',
  email: 'user@123.com',
  senha: 'user@123',
} as const;

export function isTenantDefaultUser(email: string): boolean {
  return email.trim().toLowerCase() === TENANT_DEFAULT_USER.email.toLowerCase();
}
